volatility3/framework/plugins/linux/pslist.py

```diff
@@ -81,7 +83,7 @@ def filter_func(x):
     @classmethod
     def get_task_fields(
         cls, task: interfaces.objects.ObjectInterface, decorate_comm: bool = False
-    ) -> Tuple[int, int, int, str]:
+    ) -> Tuple[int, int, int, int, str, datetime.datetime]:
```

Comment:
This change to the return response of a classmethod (the exposed API) should've meant that the MAJOR version got bumped, sorry for missing it...  5:S